corehq/apps/fixtures/tests/test_upload.py

```diff
@@ -197,6 +197,10 @@
         'things': [('UID', 'Delete(Y/N)', 'field: name'), (None, 'N', 'apple')],
         'types': [('Delete(Y/N)', 'table_id', 'is_global?', 'field 1'), ('N', 'things', 'yes', 100)]
     }),
+    ('not_excel_file', [
+        "Upload failed! Please make sure you are using a valid Excel 2007 or later (.xlsx) file. " \
+        "Error details: \"There is no item named '[Content_Types].xml' in the archive\".",
```

Comment:
I'm guessing that this isn't too useful for most people, but its probably being shown in other uploads

Reply:
Agreed the details aren't actionable in this case. I imagine they vary a lot depending on what type of not-Excel content the user uploads.